README.md

```diff
@@ -396,10 +396,10 @@ Trait `SkimItem` is provided to customize how a line could be displayed,
 compared and previewed. It is implemented by default for `AsRef<str>`
 
 Plus, `SkimItemReader` is a helper to convert a `BufRead` into
-`SkimItemReceiver` (we can easily turn a `File` for `String` into `BufRead`).
-So that you could deal with strings or files easily.
+`SkimItemReceiver` (we can easily turn a `File` or `String` into `BufRead`),
+so that you could deal with strings or files easily.
 
-Check more examples under [examples/](https://github.com/lotabout/skim/tree/master/examples) directory.
+Check out more examples under the [examples/](https://github.com/lotabout/skim/tree/master/skim/examples) directory.
```

Comment:
No `the` here I think, under <folder> sounds better

Reply:
Without `the`, we would have to remove "directory." I don't know how to explain it, it just sounds more right to me.

I feel like we can have either of these sound good. 
* "Check out more examples under the [examples/](link) directory."
* "Check out more examples under examples."

Reply:
I missed "directory", you're right